pkg/machinery/config/types/v1alpha1/v1alpha1_types.go

```diff
@@ -330,6 +330,30 @@ var (
 		DHCPRouteMetric: 1024,
 	}
 
+	networkConfigWireguardHostExample = &DeviceWireguardConfig{
+		WireguardPrivateKey: "UJmtdL+Yn6UVL9xzuU0Z9lixl5Fh3whWPMhQ/yKiZGM=",
+		WireguardListenPort: 51111,
+		WireguardPeers: []*DeviceWireguardPeer{
+			{
+				WireguardPublicKey:  "Y770LgBENm+gdlDOaf+voGHEBaFhADH6qSiqD7qMN2g=",
+				WireguardEndpoint:   "192.168.1.3",
+				WireguardAllowedIPs: []string{"192.168.1.0/24"},
+			},
+		},
+	}
+
+	networkConfigWireguardPeerExample = &DeviceWireguardConfig{
+		WireguardPrivateKey: "+AdBen89qXzYCrheNYabDZ/X0JbvUajoBl4BwChhNWU=",
+		WireguardPeers: []*DeviceWireguardPeer{
+			{
+				WireguardPublicKey:                   "Ac3M3kTLwN57uGobYizrO9jeuk4Th3v7ET101jHr7Xw=",
+				WireguardEndpoint:                    "192.168.1.2",
+				WireguardPersistentKeepaliveInterval: time.Second * 10,
+				WireguardAllowedIPs:                  []string{"192.168.1.0/24"},
```

Comment:
Should this be Wireguard network IPs or "real" IPs?

Reply:
WireguardNetwork IPs as far as I understand.